### ISSUE TITLE
Keyboard support within tagging

### ISSUE THREAD

**Issue by philfreo** [select2/select2#910]
In the examples for "Tagging Support" and similar...

I should be able to use left/right arrow keys to go "between" tags (and easily add tags in between existing ones, or backspace to remove one of them after I navigated to the right of it)

**Comment by kevin-brown** [comment]
Pushing this off from 4.0 for now, as we don't have time to implement it.

**Comment by ryantheleach** [comment]
This is the sole reason why I had to choose a different library.
